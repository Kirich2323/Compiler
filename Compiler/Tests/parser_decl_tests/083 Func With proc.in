function name() : integer;

 procedure anothername();
 begin
 end;

begin
end;