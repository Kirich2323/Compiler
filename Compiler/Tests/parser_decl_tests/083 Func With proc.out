main
----------
char                     type 
float                    type 
integer                  type 
name                     func 
 arguments:
    result            func result         integer
 locals:
    anothername              proc 
     arguments:
     locals:

